gammapy/maps/axes.py

```diff
@@ -1131,23 +1135,38 @@ def downsample(self, factor):
         if self.node_type == "edges":
             nbin = self.nbin / factor
 
-            if np.mod(nbin, 1) > 0:
-                raise ValueError(
-                    f"Number of {self.name} bins is not divisible by {factor}"
-                )
+            if strict and np.mod(nbin, 1) > 0:
+                idx = np.arange(0, self.nbin, factor)
+                if idx[-1] < self.nbin:
+                    idx = np.append(idx, self.nbin)
+                edges = self.edges[idx]
 
-            edges = self.edges[::factor]
-            return self.from_edges(edges, name=self.name, interp=self.interp)
+            else:
+                if np.mod(nbin, 1) > 0:
+                    raise ValueError(
+                        f"Number of {self.name} bins is not divisible by {factor}"
+                    )
+                edges = self.edges[::factor]
```

Comment:
Do you still need this?
```
idx = np.arange(0, self.nbin, factor)
 if idx[-1] < self.nbin:
      idx = np.append(idx, self.nbin)
 edges = self.edges[idx]
```
should suffice.
The `ValueError` can be raised inside the `if` statement `if strict is True`

Reply:
When running the tests, it appears I still need it